app/operations/base_operations.rb

```diff
@@ -56,6 +57,27 @@ def operate!
     end
   end
 
+  class UpdateAll < BaseOperations::Create
+    def initialize(model, params, current_user)
+      @params = params.to_hash
+      super(model, current_user)
+    end
+
+    def self.name
+      'update'
```

Comment:
Should this be `'update_all'`?

Reply:
We must to use like that, we have a name verification inside the `activity_recording.rb:14` for that we need to change the name of our classes to `"destroy"` instead of `"destroy_all"` for example. :grin: